### DISCUSSION TITLE
Confusing Tablet/Gamepad/MIDI enable checkboxes

### DISCUSSION THREAD

**Discussion by K4ktus123** [ppy/osu#13190] [Ideas]
After disabling Tablet, Joystick/Gamepad or MIDI, the text "Enabled" next to the checkbox doesn't change.
![image](https://user-images.githubusercontent.com/55000815/113437663-5522bb00-93e7-11eb-9616-2bc142a7a364.png)
It might be confusing at first.

How about having the text change to "Disabled" after unchecking the option?
![image](https://user-images.githubusercontent.com/55000815/113437857-b2b70780-93e7-11eb-820b-be280a1891b8.png)

Or maybe just position the checkboxes next to the texts describing them?
![image](https://user-images.githubusercontent.com/55000815/113437940-dda15b80-93e7-11eb-8b91-24258ae63986.png)

**Comment by frenzibyte** [comment]
I would agree with making the checkbox next to the input section itself, and gray everything out if disabled.

**Comment by peppy** [comment]
Shouldn't be attempted until the settings UI is refreshed. The implementation here was intentionally done in the easiest-to-implement way without design considerations.
